Jenkinsfile

```diff
@@ -31,8 +31,15 @@ pipeline {
     }
     stage('Deploy') {
       steps {
-        sh 'python3 setup.py build'
+        sh 'tar -cvzf build.tar.gz build/'
+        sh 'tar -cvzf reports.tar.gz reports/'
       }
     }
   }
-  }
\ No newline at end of file
+  post {
+    always {
+      archiveArtifacts 'reports.tar.gz'
```

Comment:
I suggest using onlyIfSuccessful: true 
With that, if CI fails, it won't make the artifact.

Reply:
I think that with reports is interesting to have a report, it may tell you what have failed, but it is an interesting idea to apply it to build

Reply:
Yes, it's better using it only with the build